src/main/java/calculator/Operand.java

```diff
@@ -0,0 +1,30 @@
+package calculator;
+
+import lombok.Getter;
+
+@Getter
+public abstract class Operand<T> {
+    protected T value;
+    protected Operator<T> former;
+    protected Operator<T> latter;
+
+    public Operand(T value){
```

Comment:
```suggestion
    protected Operand(T value){
```

abstract class의 경우 constructor의 접근 제한자를 protected로 수정하여 상속한 클래스에서만 호출이 가능하도록 하는 것이 좋을 듯 합니다.